toolkit/ThreeWToolkit/data_visualization/plot_multiple_series.py

```diff
@@ -0,0 +1,60 @@
+from typing import cast
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+from .base_visualizer import BaseVisualizer
+
+
+class PlotMultipleSeries(BaseVisualizer):
+    def __init__(
+        self,
+        series_list: list[pd.Series],
+        labels: list[str],
+        title: str,
+        xlabel: str,
+        ylabel: str,
+        **plot_kwargs,
+    ) -> None:
+        if len(series_list) != len(labels):
+            raise ValueError("series_list and labels must have the same length")
+
+        self.series_list = series_list
+        self.labels = labels
+        self.title = title
+        self.xlabel = xlabel
+        self.ylabel = ylabel
+        self.plot_kwargs = plot_kwargs
+
+    def plot(self, ax: Axes | None = None) -> tuple[Figure, Axes]:
+        if ax is None:
+            fig, ax = plt.subplots(figsize=(12, 6))
+        else:
+            fig = cast(Figure, ax.get_figure())
+
+        cmap = plt.get_cmap("Set1", len(self.series_list))
+        colors = [cmap(i) for i in range(len(self.series_list))]
+
+        plotted_any = False
+        for i, (series, label) in enumerate(zip(self.series_list, self.labels)):
+            ax.plot(
```

Comment:
Before `ax.plot` it would be good to check if series contains only NaN values, as done for another plot:

```
clean_series = series.dropna()
if clean_series.empty:
    continue
```